tools/auto-compose/dagfactory/utils.py

```diff
@@ -0,0 +1,72 @@
+import datetime
+import re
+
+
+def get_start_date(date_value):
+    """
+    Takes value from DAG config and generates valid start_date. Defaults to
+    today, if not a valid date or relative time (1 hours, 1 days, etc.)
```

Comment:
Should this default to yesterday? I recall in airflow needing to set the start_date at least one schedule interval previous to the first desired dag run.